controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -844,6 +905,70 @@ bool wfa_ca::get_send_1905_1_tlv_hex_list(std::list<tlv_hex_t> &tlv_hex_list,
     return false;
 }
 
+/**
+ * @brief Writes a decimal string into the class buffer.
+ * 
+ * @param[in] value Decimal string.
+ * @param[out] length Length of current tlv.
+ * @return false if length smaller than data, else true.
+ */
+bool tlvPrefilledData::add_tlv_value_decimal_string(const std::string &value, uint16_t &length)
+{
+    int i_dec        = std::stoi(value.substr(2, value.length()));
+    int num_of_bytes = number_of_bytes(i_dec);
```

Comment:
As I have said [here](https://github.com/prplfoundation/prplMesh/pull/313#discussion_r328571565), why didn't you use divide and modulo? 
```cpp
num_of_bytes  = i_dec / UINT8_MAX + ( i_dec % UINT8_MAX != 0 ? 0 : 1);
```
This is one simple line, and you can remove `number_of_bytes()`.